definitions/pipelines/somatic_wgs.cwl

```diff
@@ -166,6 +166,9 @@ inputs:
         type: File?
         secondaryFiles: [.tbi]
         doc: "An optional VCF with variants that will be flagged as 'VALIDATED' if found in this pipeline's main output VCF"
+    cnvkit_target_average_size:
+        type: int?
+        doc: "approximate size of split target bins for CNVkit"
```

Comment:
Is there a sensible default already set here in the tool? (it's not set in the CWL explicitly)  If so, might be worth mentioning that in the doc line here.  ("if not provided, CNVkit will use XX")